Dockerfile

```diff
@@ -35,3 +35,4 @@ RUN bash -c "if [ $DEVEL_COLLECTION_LIBRARY -ne 0 ]; then \
 RUN pip install .
 
 RUN chmod -R 0775 $APP_DIR
+RUN chmod -x $APP_DIR/tests/e2e/files/passwords/*.*
```

Comment:
@Alex-Izquierdo do you know why files are getting copied over with the execute permission?

Reply:
@Alex-Izquierdo yes. we are wondering why it is necessary to make every file executable.

Reply:
I don't think this was intended, just a quick fix. We need to revisit and apply perms with more precision.